src/types/FateSet.js

```diff
@@ -0,0 +1,29 @@
+const FateData = require('./FateData')
+const {FateTypeSet} = require('../FateTypes')
+
+class FateSet extends FateData {
+    constructor(itemsType, items = []) {
+        super('set')
+
+        this._itemsType = itemsType
+        this.items = Array.from(items)
+    }
+
+    get itemsType() {
+        return this._itemsType
+    }
+
+    get type() {
+        return FateTypeSet(this.itemsType)
+    }
+
+    get length() {
+        return this.items.length
+    }
+
+    valueOf() {
+        return this.items.map(e => e.valueOf())
```

Comment:
Shouldn't it be a JS `Set` in consistency with `FateMap`?
https://github.com/aeternity/aepp-calldata-js/blob/7b983e934b63c2179f16b262daf491ae3a2d45b2/src/types/FateMap.js#L43-L50